ios/pod_helpers.rb

```diff
@@ -98,6 +98,16 @@ def try_pod(name, podspec, project_root)
   pod name, :podspec => podspec if File.exist?(File.join(project_root, podspec))
 end
 
+def append_additional_build_settings!(build_settings, settings_to_append)
+  settings_to_append&.each do |setting, value|
+    if build_settings.key?(setting)
+      build_settings[setting] += value
```

Comment:
Can we prefix a space if the value is a string so users don't have to think about it?
```suggestion
      current = build_settings[setting]
      build_settings[setting] += " #{value}" if current.is_a? String
      build_settings[setting] += value if current.is_a? Array
```